ironfish/src/mining/pool.ts

```diff
@@ -541,4 +543,25 @@ export class MiningPool {
       await this.shares.updateBlockStatus(block, main, confirmed)
     }
   }
+
+  async updateUnconfirmedPayoutTransactions(): Promise<void> {
+    const unconfirmedTransactions = await this.shares.unconfirmedPayoutTransactions()
+
+    for (const transaction of unconfirmedTransactions) {
+      const transactionInfoResp = await this.rpc.getAccountTransaction({
+        hash: transaction.transactionHash,
+        confirmations: this.config.get('confirmations'),
```

Comment:
We rely on the confirmations of the node for blocks, but use the confirmation of the pool here. This isn't a big deal just pointing out should we just delete this and use the nodes confirmations config too?